.github/workflows/dependency-updates-post-processing.yaml

```diff
@@ -94,7 +92,9 @@ jobs:
             git commit --message="${tidy_message}"
             git pull --rebase=true
             git push
-            gh pr review --approve
+            if [ "${{ github.actor }}" == "renovate[bot]" ]; then
+              gh pr review --approve
+            fi
```

Comment:
Thanks for highlighting this! This was only needed during the time period when we had "Dismiss stale PR reviews" on, as this commit would dismiss the previous approval. Now we can remove it again.

```suggestion
```

Reply:
thanks for this background info! i adjusted this with da634f9